wpi_interface/src/main/java/coppercore/wpi_interface/Controllers.java

```diff
@@ -0,0 +1,174 @@
+package coppercore.wpi_interface;
+
+import coppercore.parameter_tools.JSONExclude;
+import coppercore.parameter_tools.JSONSync;
+import edu.wpi.first.wpilibj2.command.button.CommandGenericHID;
+import edu.wpi.first.wpilibj2.command.button.CommandJoystick;
+import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
+import edu.wpi.first.wpilibj2.command.button.Trigger;
+import java.util.List;
+import java.util.Map;
+import java.util.function.DoubleSupplier;
+import java.util.function.IntSupplier;
+
+public class Controllers {
+    public Map<String, Integer> buttonShorthands;
+    public Map<String, Integer> axesShorthands;
+    public List<Controller> controllers = null;
+
+    @JSONExclude
+    public static JSONSync<Controllers> synced =
+            new JSONSync<Controllers>(
+                    new Controllers(), "filePath", new JSONSync.JSONSyncConfigBuilder().build());
```

Comment:
Can we provide a way to pass in a path instead of having it hardcoded? This way you don't need a placeholder here, because we don't really have a way to replace the placeholder unless I'm missing something here.

Reply:
You can call Controllers.synced.setFile(<newFilePath>) and I will overload the loading controllers function to allow a file path to be set

Reply:
Awesome, that sounds good.